Alignment/MuonAlignment/src/MuonAlignmentInputXML.cc

```diff
@@ -267,7 +267,7 @@ AlignableMuon *MuonAlignmentInputXML::newAlignableMuon(const edm::EventSetup& iS
    XercesDOMParser *parser = new XercesDOMParser();
    parser->setValidationScheme(XercesDOMParser::Val_Always);
 
-   ErrorHandler *errHandler = (ErrorHandler*)(new HandlerBase());
+   xercesc_3_1::ErrorHandler *errHandler = (xercesc_3_1::ErrorHandler*)(new HandlerBase());
```

Comment:
@smuzaffar from inspecting the file and other parts of CMSSW I think the correct string would be `XERCES_CPP_NAMESPACE` instead of `xercesc_3_1`